### DISCUSSION TITLE
[FEATURE] ...

### DISCUSSION THREAD

**Discussion by the-grizzly-bear** [projectdiscovery/nuclei#6576] [General]
### Describe your feature request

After installing nuclei is never executable, add install instructions to add to path.

### Describe the use case of the feature

Add to the readme for the install section:

go install -v github.com/projectdiscovery/nuclei/v3/cmd/nuclei@latest
echo $GOPATH
export PATH=$PATH:$HOME/go/bin

**Comment by GeorginaReeder** [comment]
Thanks so much for your feature request @spicy-bear . We'll take a look into this!

**Comment by Mzack9999** [comment]
This is already automated if you install nuclei with [pdtm](https://github.com/projectdiscovery/pdtm)
